cfg/mounts.kv

```diff
@@ -13,81 +16,245 @@
 	//			"dir"	"dir_inside" // mounts folder: GameDir\mod_folder\dir_inside
 	//		}
 	//	}
+	// =========================================================================================
 
-	// CS:GO
-	// IF YOU WANT TO MOUNT CSGO CONTENT (uncomment this for the chickens): UNCOMMENT (remove the double slashes //) OF THE LINES BETWEEN THE ASTERICKS. Do NOT remove the double slashes from the line with the astericks (*)
-	//**********************
+	// Uncomment the lines between the asterisks surrounding any Valve game's mount dirs to mount it.
+	// Do not remove the double-slashes from the lines containing asterisks, as it will break your mounts.kv file.
+
+	// Counter-Strike: Global Offensive (AppID 730)
+	// **********************
 	// 730
 	// {
-	//	"csgo"
+	// 	"csgo"
+	// 	{
+	// 		"vpk"	"pak01"
+	// 		"dir"	"download"
+	// 	}
+	// }
+	// **********************
+
+	// Half-Life 2: Episode Two (AppID 420)
+	// **********************
+	 420
+	 {
+		"ep2"
+		{
+			"vpk" "ep2_pak"
+		}
+	
+	//	// Needed for Ep2 to work properly as a standalone game mount;
+	//	// not mounting ep1's folder + vpks will result in missing assets
+	// 	// persisting across the various ep2_outland maps.
+	//	//
+		// You do not need to mount AppID 380 (Half-Life 2: Episode One)
+		// if you have Ep2 installed, as both games share content between each other.
+		"episodic"
+		{
+			"vpk"	"ep1_pak"	
+		}
+	 }
+	// **********************
+
+	// Half-Life 2: Episode Two (AppID 380)
+	// Refer to the above if you have Ep2 installed
+	// **********************
+	// 380
+	// {
+	//	"episodic"
 	//	{
-	//		"vpk"	"pak01"
-	//		"dir"	"download"
+	//		"vpk" "ep1_pak"
+	//	}
+	// }
+	// **********************
+
+	// Half-Life 2: Lost Coast (AppID 340)
+	// **********************
+	// 340
+	// {
+	//	"lostcoast"
+	//	{
+	//		"vpk" "lostcoast_pak"
+	//	}
+	// }
+	// **********************
+
+	// Team Fortress 2 (AppId 440)
+	// Known to override a few hl2 textures with tf2 counterparts, e.g.: hl2 shotgun
+	// worldmodel and concrete walls used across spytech environments
+	// **********************
+	 440
```

Comment:
This needs to be commented out.

Reply:
Yeah just realized I copied from my p2ce install